docs/configuration/configuring-profiles.md

```diff
@@ -0,0 +1,226 @@
+# Configuring profiles
+
+Profiles are the units, where you can configure what you want to poll and then assign them to the device. The definition of profile lays in `values.yaml` file
+under the `scheduler` section.
+
+Here is the instruction of how to use profiles: [Update Inventory and Profile](../deployment-configuration/#update-inventory-and-profile). 
+
+There are two types of profiles in general:
+
+1. Static profile - polling starts when profile is added to `profiles` field in `inventory` of the device
+2. Smart profile - polling starts when configured conditions are fulfilled, and the device to poll from has `smart_profiles` enabled in inventory.
+Smart profiles are useful when we have many devices of certain kind, and we don't want to configure all of them "one by one" with static profiles.
+   In order to do so, we elect one of the fields (most commonly `sysDescr`), set the filter to match all the devices of this kind and setup polling of the profile.
```

Comment:
I would suggest sth like this here: 
In order to configure smart profile do the following:
1. Choose one of the fields  polled from device, most commonly `sysDescr` is being used
2. Set the filter to match all the devices of this kind 
3. Setup polling of the profile by enabling smart profiles for devices you want this profile to be polled